### PR TITLE
MAISTRA-581 Make CNI pod resource requests & limits configurable

### PR DESCRIPTION
**Pull request by luksa**

### CODE REVIEW

File: helm/istio/charts/istio_cni/templates/daemonset.yaml [hunk 1]
```diff
@@ -98,12 +98,16 @@ spec:
             - mountPath: /host/etc/cni/
               name: etc-cni-dir
           resources:
+{{- if .Values.resources }}
+{{ toYaml .Values.resources | indent 12 }}
+{{- else }}
             requests:
```
Comment (jwendell) [inline helm/istio/charts/istio_cni/templates/daemonset.yaml:104]: I think it doesn't make sense to check for the global values as well, right?

Comment (luksa) [reply helm/istio/charts/istio_cni/templates/daemonset.yaml:104]: Yes, IMHO it does _not_ make sense to use global resource requests/limits values anywhere. Elsewhere, we are using global values, but here I just hardcoded the values that work on 4.1. Maybe we should increase them to 100Mi just to leave a bit more room.

Comment (jwendell) [reply helm/istio/charts/istio_cni/templates/daemonset.yaml:104]: +1 on increasing to 100

Comment (luksa) [reply helm/istio/charts/istio_cni/templates/daemonset.yaml:104]: OK, I've increased it to 100Mi. We can reduce it when we review the resource requests/limits of all the components.

**Review by jwendell — APPROVED** [review]

**Pull request merged**